cmd/scheduler/options/options.go

```diff
@@ -79,10 +83,14 @@ func New(origArgs []string) *Options {
 	fs.StringVar(&opts.PlacementAddress, "placement-address", "", "Addresses for Dapr Actor Placement service")
 	fs.StringVar(&opts.Mode, "mode", string(modes.StandaloneMode), "Runtime mode for Dapr Scheduler")
 
-	fs.StringVar(&opts.EtcdID, "id", "dapr-scheduler-server-0", "Scheduler server ID")
+	fs.StringVar(&opts.Id, "id", "dapr-scheduler-server-0", "Scheduler server ID")
```

Comment:
I changed this because we can have scheduler instances without a running etcd; it would feel weird to have an "etcdID" for them.